### PR TITLE
Add Build-in dataset BasicMotions for time series classify

### PR DESCRIPTION
**Pull request by a10210532**

### CODE REVIEW

File: paddlets/datasets/repository/_data_config.py [hunk 1]
```diff
@@ -122,3 +122,29 @@
         "freq": 1
     }
 )
+
+# 9> BasicMotions_Test.csv
+BasicMotionsTestDataset = DatasetConfig(
+    name = "BasicMotions_Test",
+    type = "classify",
```
Comment (yangs16) [inline paddlets/datasets/repository/_data_config.py:129]: ‘classify'--> 'classification'，是不是更好点？和'forecasting‘保持一致

File: paddlets/datasets/repository/_data_config.py [hunk 2]
```diff
@@ -122,3 +122,29 @@
...
+        "group_id": "7", 
+        "target_cols": ["0", "1", "2", "3", "4", "5"], 
+        "static_cov_cols": ["label"]
+    }
+)
+
+# 10> BasicMotions_Train.csv
+BasicMotionsTrainTDataset = DatasetConfig(
+    name = "BasicMotions_Train",
+    type = "classify",
```
Comment (yangs16) [inline paddlets/datasets/repository/_data_config.py:142]: 同上

**Review by yangs16 — COMMENTED** [review]
Some minor modifications might be made.

File: paddlets/datasets/repository/_data_config.py (hunk 1 above)
Comment (a10210532) [reply paddlets/datasets/repository/_data_config.py:129]: 收到~

**Pull request closed without merging**